### ISSUE TITLE
Move/modify event to another moment by dragging

### ISSUE THREAD

**Issue by jpggithub** [xsoh/Etar-Calendar#80]
Hi 
Thank you for this great open source app!

It would be very useful to move some event by dragging them to another hour/day/week/month instead of edit them.

It would be also useful to drag the beginning or the end of an event to reduce/enlarge its duration.

**Comment by Primokorn** [comment]
Yeah, really good idea.
Hope to see this feature in a near future.
#87 should be closed IMO

**Comment by mueller-ma** [comment]
When you are in "Month" view, it will be very unprecise. And long press on a day is already reserved for creating a new event.

**Comment by Riamse** [comment]
@mueller-ma But Google Calendar manages to do long press as well as dragging events, if I recall correctly?

Any updates on this? Does Etar aim for some feature parity with Google Calendar or is it going in its own direction?

**Comment by IzzySoft** [comment]
@Primokorn I disagree with closing the copy issue. That's what I use most (in Business Calendar), see my comment there.

But full ack to wanting drag'n'drop to move an event – moving is what I'd expect to happen when dragging (unless there's a popup in the end asking me whether to copy or move, then it's of course fine for both).

@mueller-ma when in month view, dragging should just change the date (you cannot drop it on a precise time then). In week and in day view, change date and time.

**Comment by mjray** [comment]
Drag to reschedule would be a killer feature. I have not seen any of the free/open-source calendars have it yet.

Long press may currently be used to create an event, but there are other quick ways to create events and currently no quick way to move an event, so create should give way to moving if the long press is done on an existing event.

**Comment by mjray** [comment]
> I think this is the one thing ETar is missing that makes it a completely perfect replacement for me from another certain closed-source calendar app I'm using.

I've switched to "Simple Calendar" purely to get this feature. I use it a lot. I would love to have the choice of using ETar again.

**Comment by ASAJFM** [comment]
This is an incredibly useful feature, why don't we have it yet?!

It's been 3 years since the previous comment :(

**Comment by xtofr** [comment]
Plus one 😊
